### ISSUE TITLE
Provide support for multilingual text display

### ISSUE THREAD

**Issue by blalbrit** [sul-dlss/exhibits#444]
For instance: support switching between Italian text and English text - both within the curator-supplied text of a page and the metadata labels.

**Comment by blalbrit** [comment]
- [ ] add toggle to switch between languages (language selector)
- [ ] enable curators to provide translations for narrative text
- [ ] add i18n file for UI translations
- [ ] https://github.com/sul-dlss/exhibits/issues/952 (tag translation ticket)

**Comment by ggeisler** [comment]
We decided that some of this work should be done in the Spotlight repo and so we've made new tickets there that cover much of what @blalbrit outlined in his comment above. So I'm closing this ticket to prevent potential confusion. I've also closed the Exhibits subtickets mentioned above that now have equivalent tickets in Spotlight. 

See https://github.com/projectblacklight/spotlight/issues/1912 for the story ticket in Spotlight, equivalent this this one, that lists the subtickets related to the feature. 

Of the tickets @blalbrit listed above, one does remain open in Exhibits (more appropriate here, I think): https://github.com/sul-dlss/exhibits/issues/952
